packages/rainbowkit/src/wallets/Wallet.ts

```diff
@@ -70,23 +70,34 @@ export type Wallet<C extends Connector = Connector> = {
     linux?: string;
     desktop?: string;
   };
-  hidden?: (args: {
-    wallets: {
-      id: string;
-      connector: Connector;
-      installed?: boolean;
-      name: string;
-    }[];
-  }) => boolean;
-  createConnector: () => RainbowKitConnector<C>;
-};
+  hidden?: () => boolean;
+  createConnector: (options: WalletOptionsParams) => CreateConnectorFn;
+} & RainbowKitConnector;
```

Comment:
Wagmi team told me if we want to specify our own download links and custom options we would have to pass it in like this:

```
createConnector((config) => ({
    ...injected(injectedConfig)(config),
    ...walletOptions // Here all our configs (e.g download links, icon url and etc...)
}));
```